pattern-1/conf/mysql/dbscripts/is_user-mgt.sql

```diff
@@ -212,6 +213,267 @@ CREATE TABLE UM_PROFILE_CONFIG(
             PRIMARY KEY (UM_ID, UM_TENANT_ID)
 )ENGINE INNODB;
 
+CREATE TABLE IF NOT EXISTS UM_CLAIM_BEHAVIOR(
+    UM_ID INTEGER NOT NULL AUTO_INCREMENT,
+    UM_PROFILE_ID INTEGER,
+    UM_CLAIM_ID INTEGER,
+    UM_BEHAVIOUR SMALLINT,
+    UM_TENANT_ID INTEGER DEFAULT 0,
+    FOREIGN KEY(UM_PROFILE_ID, UM_TENANT_ID) REFERENCES UM_PROFILE_CONFIG(UM_ID,UM_TENANT_ID),
+    FOREIGN KEY(UM_CLAIM_ID, UM_TENANT_ID) REFERENCES UM_CLAIM(UM_ID,UM_TENANT_ID),
+    PRIMARY KEY(UM_ID, UM_TENANT_ID)
+)ENGINE INNODB;
+
+CREATE TABLE UM_HYBRID_ROLE(
+            UM_ID INTEGER NOT NULL AUTO_INCREMENT,
+            UM_ROLE_NAME VARCHAR(255),
+            UM_TENANT_ID INTEGER DEFAULT 0,
+            PRIMARY KEY (UM_ID, UM_TENANT_ID)
+)ENGINE INNODB;
+
+CREATE TABLE UM_HYBRID_USER_ROLE(
+            UM_ID INTEGER NOT NULL AUTO_INCREMENT,
+            UM_USER_NAME VARCHAR(255),
+            UM_ROLE_ID INTEGER NOT NULL,
+            UM_TENANT_ID INTEGER DEFAULT 0,
+            UM_DOMAIN_ID INTEGER,
+            UNIQUE (UM_USER_NAME, UM_ROLE_ID, UM_TENANT_ID, UM_DOMAIN_ID),
+            FOREIGN KEY (UM_ROLE_ID, UM_TENANT_ID) REFERENCES UM_HYBRID_ROLE(UM_ID, UM_TENANT_ID) ON DELETE CASCADE,
+            FOREIGN KEY (UM_DOMAIN_ID, UM_TENANT_ID) REFERENCES UM_DOMAIN(UM_DOMAIN_ID, UM_TENANT_ID) ON DELETE CASCADE,
+            PRIMARY KEY (UM_ID, UM_TENANT_ID)
+)ENGINE INNODB;
+
+CREATE TABLE UM_SYSTEM_ROLE(
+            UM_ID INTEGER NOT NULL AUTO_INCREMENT,
+            UM_ROLE_NAME VARCHAR(255),
+            UM_TENANT_ID INTEGER DEFAULT 0,
+            PRIMARY KEY (UM_ID, UM_TENANT_ID)
+)ENGINE INNODB;
+
+CREATE INDEX SYSTEM_ROLE_IND_BY_RN_TI ON UM_SYSTEM_ROLE(UM_ROLE_NAME, UM_TENANT_ID);
+
+CREATE TABLE UM_SYSTEM_USER_ROLE(
+            UM_ID INTEGER NOT NULL AUTO_INCREMENT,
+            UM_USER_NAME VARCHAR(255),
+            UM_ROLE_ID INTEGER NOT NULL,
+            UM_TENANT_ID INTEGER DEFAULT 0,
+            UNIQUE (UM_USER_NAME, UM_ROLE_ID, UM_TENANT_ID),
+            FOREIGN KEY (UM_ROLE_ID, UM_TENANT_ID) REFERENCES UM_SYSTEM_ROLE(UM_ID, UM_TENANT_ID),
+            PRIMARY KEY (UM_ID, UM_TENANT_ID)
+)ENGINE INNODB;
+
+
+CREATE TABLE UM_HYBRID_REMEMBER_ME(
+            UM_ID INTEGER NOT NULL AUTO_INCREMENT,
+            UM_USER_NAME VARCHAR(255) NOT NULL,
+            UM_COOKIE_VALUE VARCHAR(1024),
+            UM_CREATED_TIME TIMESTAMP,
+            UM_TENANT_ID INTEGER DEFAULT 0,
+            PRIMARY KEY (UM_ID, UM_TENANT_ID)
+)ENGINE INNODB;
+
+USE WSO2_IDENTITY_DB;
```

Comment:
Any specific reason to switch to use database within script.
Previous way is better so if database needs to be changed that can be done at one point.
Moreover schema definition is duplicated in the sql script

Reply:
Fixed